kano_registration_gui/GetData.py

```diff
@@ -180,17 +192,11 @@ def get_cached_username_and_birthday(self):
         }
 
     def widgets_full(self, widget=None, event=None):
-        full = True
-
-        if not self.username_entry.validated:
-            full = False
-
-        if not self.password_entry.validated:
-            full = False
-
-        bday_filled = self.bday_widget.birthday_entries_filled()
+        if (
```

Comment:
When breaking lines after an if, use twice the indentation, to ensure that the block following the ':' is easily distinguished.

```python
if (width == 0 and height == 0 and
        color == 'red' and emphasis == 'strong' or
        highlight > 100):
    raise ValueError("sorry, you lose")
```